code/hsec-tools/src/Security/Advisories/Parse.hs

```diff
@@ -5,53 +5,48 @@
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE LambdaCase #-}
-
+{-# OPTIONS_GHC -Wno-orphans #-}
 module Security.Advisories.Parse
   ( parseAdvisory
   , OutOfBandAttributes(..)
   , emptyOutOfBandAttributes
   , AttributeOverridePolicy(..)
   , ParseAdvisoryError(..)
-  , TableParseError(..)
   )
   where
 
-import Control.Monad ((>=>))
 import Data.Bifunctor (first)
 import Data.Foldable (toList)
-import Data.Functor.Identity (Identity(Identity))
-import Data.List.NonEmpty (NonEmpty(..))
 import Data.Maybe (fromMaybe)
 import Data.Monoid (First(..))
 import Data.Tuple (swap)
 import GHC.Generics (Generic)
 
-import Control.Monad.Except
-  ( ExceptT (ExceptT)
-  , MonadError
-  , throwError
-  )
 import qualified Data.Map as Map
 import Data.Sequence (Seq((:<|)))
-import qualified Data.Set as Set
 import qualified Data.Text as T
 import qualified Data.Text.Lazy as T (toStrict)
-import Data.Time (LocalTime(..), ZonedTime(..), midnight, utc)
+import Data.Time (ZonedTime(..), LocalTime (LocalTime), midnight)
 import Distribution.Parsec (eitherParsec)
 import Distribution.Types.VersionRange (VersionRange)
 
 import Commonmark.Html (Html, renderHtml)
 import qualified Commonmark.Parser as Commonmark
 import Commonmark.Types (HasAttributes(..), IsBlock(..), IsInline(..), Rangeable(..), SourceRange(..))
 import Commonmark.Pandoc (Cm(unCm))
-import qualified TOML
+import qualified Toml
+import qualified Toml.FromValue as Toml
+import qualified Toml.FromValue.Matcher as Toml
+import qualified Toml.ToValue as Toml
 import Text.Pandoc.Builder (Blocks, Many(..))
 import Text.Pandoc.Definition (Block(..), Inline(..), Pandoc(..))
 import Text.Pandoc.Walk (query)
 import Text.Parsec.Pos (sourceLine)
 
 import Security.Advisories.Definition
 import Security.OSV (Reference(..), referenceTypes)
+import Data.List (intercalate)
+import Data.Time.LocalTime (utc)
```

Comment:
This is the same as `Data.Time (utc)`, which was removed from the export list above.  I prefer to keep it as it was and avoid this additional `import` line.